lib/screens/events/events_repository.dart

```diff
@@ -47,8 +47,8 @@ class FakeEventsRepository implements EventsRepository {
           }
         ]
       };
-      List<Event> events;
-      (json["data"] as List).map((e) => events.add(Event.fromJson(e)));
+      List<Event> events = List();
+      (json["data"] as List).map((e) => events.add(Event.fromJson(e))).toList();
```

Comment:
🔴 
`map` is different from `forEach`. Please read about them.
1. Map
```suggestion
      List<Event> events = (json["data"] as List).map((e) => events.add(Event.fromJson(e))).toList();
```
2. forEach
```suggestion
      List<Event> events = List();
      (json["data"] as List).forEach((e) => events.add(Event.fromJson(e)));
```